rskj-core/src/main/java/co/rsk/net/BlockProcessResult.java

```diff
@@ -48,7 +48,7 @@ public BlockProcessResult(boolean additionalValidations, Map<ByteArrayWrapper, I
     }
 
     public boolean wasBlockAdded(Block block) {
-        return additionalValidationsOk && !result.isEmpty() && importOk(result.get(new ByteArrayWrapper(block.getHash())));
+        return additionalValidationsOk && !result.isEmpty() && importOk(result.get(new ByteArrayWrapper(block.getHash().getBytes())));
```

Comment:
"importOk(result.get(new ByteArrayWrapper(block.getHash().getBytes())))"

we can do better !!